src/components/select-field/SelectField.scss

```diff
@@ -29,3 +29,15 @@
     max-height: 340px; // show at most 12 items in the dropdown menu
     overflow-y: auto;
 }
+
+.bdl-SelectFieldDropdown {
```

Comment:
You can choose to have this as bdl-SelectField-dropdown too considering the drop-down component is not to be used on its own. We do it for other components too that span file boundaries.

Reply:
I think I'll keep the same as I'm not entirely sure of @ConradJChan's intent